code/modules/mob/living/simple_animal/abnormality/teth/meat_lantern.dm

```diff
@@ -121,7 +121,7 @@
 	..()
 
 /mob/living/simple_animal/hostile/abnormality/meat_lantern/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
-	if(work_time < 18 SECONDS)
+	if(work_time < 18 SECONDS && get_attribute_level(user, TEMPERANCE_ATTRIBUTE) <= 60)
```

Comment:
similar thing to my last comment, is it supposed to be if yo work too fast WHILE having bad temperance?